sdv/logging/logger.py

```diff
@@ -35,7 +59,7 @@ def get_sdv_logger(logger_name):
             logger.removeHandler(handler)
 
         if logger_name in logger_conf.get('loggers'):
-            formatter = None
+            formatter = CSVHandler()
```

Comment:
I think that this is not a `formatter` but a `handler`.
I believe that we could do something like this:
```python
if handler_class == 'CSVHandler':
    # Do the logic in here
```

Reply:
We're still using the FileHandler as our handler since we're writing to the file. The `CSVHandler` (to be renamed CSVFormatter) just formats the log message before passing it along to the handler, in this case converting it to a CSV row. We can add it to the config though, or case it on the file target extension.